{"name": "输入错误密码登录失败", "status": "passed", "attachments": [{"name": "stdout", "source": "f4c7e6b8-6b50-4d12-ab2e-2d041d10372d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '5', 'casename': '输入错误密码登录失败', 'mode': '登录', 'data': 'userAccount=979172251@qq.com\\npassword=a1234567', 'assert_way': 'class name=login-form-forgot', 'result': '\\n              Forgot Password?\\n            '}"}], "start": 1651036235254, "stop": 1651036239877, "uuid": "e9ef9228-b1df-45f3-a705-6d187ec0be29", "historyId": "104e3d5feb09afaca2b374c9a621f082", "testCaseId": "f42af1ba93a36a6b78bc4dd2bc5b083c", "fullName": "test_main.Test_Main#test_login", "labels": [{"name": "feature", "value": "登录"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "21748-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}